### PR TITLE
Doc: pooch install for xarray dataset

### PR DESCRIPTION
**Pull request by Ishaanj18**
[As specified here](https://github.com/observingClouds/xbitinfo/pull/184#issuecomment-1495225718)

### CODE REVIEW

File: docs/quick-start.ipynb [hunk 1]
```diff
@@ -29,11 +29,39 @@
    "source": [
     "ds = xr.tutorial.load_dataset(\"eraint_uvz\").astype(\"float32\")\n",
     "\n",
-    "xb.plot_distribution(ds)",
+    "xb.plot_distribution(ds)\n",
```
Comment (observingClouds) [inline docs/quick-start.ipynb:32]: I think this got in here by accident. Could you undo this to the original version?

File: docs/quick-start.ipynb [hunk 2]
```diff
@@ -61,7 +89,7 @@
    "id": "bdfd9186",
    "metadata": {},
    "source": [
-    "## Visualize information content",
+    "## Visualize information content\n",
```
Comment (observingClouds) [inline docs/quick-start.ipynb:92]: Same as above

**Review by observingClouds — CHANGES_REQUESTED** [review]
If you could undo the two additions of `\n` we should be good to merge this PR.

**Comment by ayoubft** [comment]
Dear @Ishaanj18 ,

I am writing to express my frustration regarding this pull request.  I made an effort to start the other PR, and **observingClouds** suggested adding this to the docs. I wanted to implement that, only to find that you had already made those changes.

While I understand that you are keen to contribute, and take the initiative, but not like this. Similar behavior can make it for starters like me very difficult to feel motivated to contribute.

I believe that collaborative work should involve clear communication and mutual respect.

**Comment by Ishaanj18** [comment]
Really sorry for the inconvenience that I have caused, even I am new to open source contribution. It has been pointed out to me now that the cooldown period for open source is 2 weeks. I made the request as there had been no updates/changes made on the prior PR and had been marked as closed. I will keep this in mind in future.
Looking forward to collaborating with you.
Regards

**Comment by ayoubft** [comment]
Hi @Ishaanj18,
No worries, I understand. The issue was not yet stale, but I appreciate your apology. I'm looking forward to collaborating with you too.
Thanks

**Review by observingClouds — APPROVED** [review]

**Pull request merged**